src/Resources/config/services/api.xml

```diff
@@ -6,18 +6,21 @@
         <parameter key="sylius.paypal.logging_level_increased">true</parameter>
     </parameters>
     <services>
+        <service id="Http\Discovery\Psr18Client" class="Http\Discovery\Psr18Client" />
         <service
             id="Sylius\PayPalPlugin\Client\PayPalClientInterface"
             class="Sylius\PayPalPlugin\Client\PayPalClient"
         >
-            <argument type="service" id="sylius.http_client" />
+            <argument type="service" id="Http\Discovery\Psr18Client" />
```

Comment:
This change is probably not needed as `sylius.http_client` is an alias of `Psr\Http\Client\ClientInterface` in 1.13 🤔 if so, it should be applied to all changed arguments

Reply:
I've verified this. In Sylius 1.12 `sylius.http_client` is an alias of `GuzzleHttp\Client`. We have to go with `Psr\Http\Client\ClientInterface` because we will have deprecations in 1.12